html/Mogo/js/sections-animation/helper-functions.js

```diff
@@ -0,0 +1,45 @@
+import {components} from './menu-elements.js';
+
+const {navbar, navbarMenu} = components;
+
+function setNavbarBackground() {
+  navbar.classList.add('navbar--with-bg');
+}
+
+function removeNavbarBackground() {
+  navbar.classList.remove('navbar--with-bg');
+}
+
+function changeNavbarBackground() {
+  if (window.scrollY >= 200) {
+    setNavbarBackground();
+  } else if (!navbarMenu.classList.contains('navbar__menu--active')) {
+    removeNavbarBackground();
+  }
+}
+
+function addScrollEffect(event) {
+  const elementName = event.getAttribute('href');
+  const element = document.querySelector('#' + elementName);
+  window.scrollTo({
```

Comment:
just in case, this method is awesome, but don't have good support and probably you will need to do it in some different way, don't have to change it